### PR TITLE
feat: add run logs and adapt get status

### PR DESCRIPTION
**Pull request by jupyterjazz**
- Implement getting run logs
- Adapt getting run status
- Bump hubble client version

~~Integration tests are failing because we use old api version on the server.~~

### CODE REVIEW

**Comment by gmastrapas** [comment]
Where did you update the get run status?

**Comment by jupyterjazz** [comment]
> Where did you update the get run status?

https://github.com/jina-ai/finetuner/blob/b5456cc4b124e63b2947ac6ec95e7a5af12ae125/finetuner/run.py#L48

File: finetuner/run.py [hunk 1]
```diff
@@ -45,14 +45,16 @@ def status(self) -> str:
         """
         return self._client.get_run_status(
             experiment_name=self._experiment_name, run_name=self._name
-        )
+        )[STATUS]
```
Comment (gmastrapas) [inline finetuner/run.py:48]: I would say return the full dict, the details are useful

**Review by gmastrapas — APPROVED** [review]
LGTM!

**Pull request merged**